examples/monitoring/openai_proxy_quickstart_enterprise_mode.ipynb

```diff
@@ -0,0 +1,280 @@
+    "%%bash\n",
+    "curl \"$OPENAI_PROXY_API_BASE/chat/completions\" \\\n",
+    "-H \"Authorization: Bearer $OPENAI_PROXY_API_KEY\" \\\n",
+    "-H \"Content-Type: application/json\" \\\n",
+    "-H \"X-Wandb-Client-Id: Shakespearean-Gradients-001\" \\\n",
+    "-H \"X-Wandb-Attribute-Persona: Shakespeare\" \\\n",
+    "-d '{\n",
+    "     \"model\": \"gpt-3.5-turbo\",\n",
+    "     \"messages\": [{\"role\": \"user\", \"content\": \"Tell me a joke about loss functions if you were Shakespeare!\"}]\n",
+    "   }'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Admins can view logs at {WEAVE_ROOT_URL}/browse/wandb/[ADMIN_ENTITY]/monitoring/openai\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "source": [
+    "## Use Proxy with Weave SDK\n",
+    "\n",
+    "Now, users will typically want to track their own logs so they can analyze them independently. In this case we combine the monitoring techiques discussed in [OpenAI Monitoring Demo](./openai_client_quickstart.ipynb) with the proxy concepts here.\n",
+    "\n",
+    "You can use this proxy with the Weave SDK to log out to a personal streamtable and a proxy-configured streamtable simultaneously.  This can be useful for admins who want to monitor API calls in their organization.\n",
+    "- Users specify a personal table path with `init_monitor(entity/project/table)`.  Any `ChatCompletion.create` calls with automatically log to this table.\n",
+    "- Then, if the `openai.api_base` and `openai.api_key` are set to proxy format, the above calls will also log the proxy-configured table.\n",
+    "\n",
+    "NOTE: This currently does not work with streaming `ChatCompletion`, but we hope to support it soon!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "WB_ENTITY = # FILL OUT\n",
+    "WB_PROJECT = \"llmon\"\n",
+    "STREAM_NAME = \"openai_logs\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install weave tiktoken"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"WANDB_BASE_URL\"] = WANDB_BASE_URL\n",
+    "import wandb\n",
+    "wandb.login()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openai\n",
+    "from weave.monitoring import openai as openaimon\n",
+    "from weave.monitoring import init_monitor\n",
+    "\n",
+    "m = init_monitor(f\"{WB_ENTITY}/{WB_PROJECT}/{STREAM_NAME}\")\n",
+    "\n",
+    "r = openaimon.ChatCompletion.create(model=\"gpt-3.5-turbo\", messages=[{\"role\": \"user\", \"content\": f\"hello world!\"}])\n",
```

Comment:
I think if we want to get super fancy, we could add some attributes here and maybe add a third call? right now with two calls i don't get bars in the top panel. but also maybe for later!